{{cookiecutter.project_slug}}/.dependabot/config.yml

```diff
@@ -0,0 +1,6 @@
+version: 1
+update_configs:
+  - package_manager: "python"
+    directory: "/"
+    update_schedule: "live"
+    
```

Comment:
fix these line endings

Reply:
How do you want them fixed?

Reply:
Remove trailing lines and make existing lines match this specification: https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206